src/UKHO.S100PermitService.Common/Encryption/AesEncryption.cs

```diff
@@ -25,6 +38,19 @@ public string Decrypt(string hexString, string hexKey)
             return PerformCryptography(hexString, decrypt);
         }
 
+        /// <summary>
+        /// Get encrypted data.
+        /// </summary>
+        /// <remarks>
+        /// Encrypt data using secret key.
+        /// Advanced Encryption Standard (AES) block cipher algorithm is used.This is a symmetric-key algorithm. This means that the same key is used for encryption and decryption.
+        /// Cipher Block Chaining(CBC) mode of operation and No padding is used.
+        /// For S-100 size of data and secret key is fixed to 128 bits (32 characters) hexadecimal digits, if validation fails then AesEncryptionException exception handler will be trigger.
+        /// </remarks>
+        /// <param name="hexString">Data to be encrypt.</param>
+        /// <param name="keyHexEncoded">Secret Key.</param>
+        /// <returns>Encrypted data.</returns>
+        /// <exception cref="AesEncryptionException">AesEncryptionException exception handler triggers when length validation fails.</exception>
```

Comment:
Not an exception handler, that's a catch{} block. The exception is thrown.

Reply:
Resolved